README.md

```diff
@@ -196,6 +196,33 @@ const catWatch = new ServiceClient({
 });
 ```
 
+#### Scheduled retries
+
+Timeouts are often the reason for retries. You might want to retry your request *without* waiting for the initial request to timeout, giving more time to the service you are calling to reply to initial request but still keeping the same overal requests timeout.
+
+For this purpose you can use `retryAfter` in `retryOptions` in combination with `dropAllRequestsAfter` in options root. If you have retries configured and initial request has not resolved before `retryAfter`, another request will be sent and whichever requests will resolve first will be treated, aborting the other one(s).
+
+`retryAfter` logic is immediate and will not take into account minTimeout or maxTimeout.
```

Comment:
Ah, so this one doesn't use the exponential backoff logic. Is there a specific reason not to do it?

Reply:
Only for `scheduled` retry - there is no backoff because it defeats the purpose of scheduled retry. If we get a backend response in error, we backoff as usual. The only difference is, without a response, we shoot the next request at the scheduled time.

Note that `retryAfter` works with regular retries in parallel and do not affect their logic and they share the same total amount of retries. So if you retried because of a 500 error, and had retries:2 - you will not send the scheduled retry.

Reply:
Aha, so `retryAfter` is an enough interval not to DDoS backend services. That makes sense.